contracts/test/upgrade-test.ts

```diff
@@ -95,14 +95,14 @@ describe("Upgrade", async function () {
     const walletAddress = walletOldVg.address;
     const blsSecret = walletOldVg.privateKey;
 
-    const wallet = await BlsWalletWrapper.connect(
-      blsSecret,
-      fx.verificationGateway.address,
-      fx.verificationGateway.provider,
-    );
     // Sign simple address message
     const addressMessage = solidityPack(["address"], [walletAddress]);
-    const addressSignature = wallet.signMessage(addressMessage);
+    const walletNewVg = await BlsWalletWrapper.connect(
+      blsSecret,
+      vg2.address,
+      vg2.provider,
+    );
+    const addressSignature = walletNewVg.signMessage(addressMessage);
```

Comment:
We need the message to be signed with the appropriate verification gateway now which is why I created a new wallet wrapper with the new VG